src/write.c

```diff
@@ -1358,6 +1383,154 @@ static size_t avifEncoderFindExistingChunk(avifRWStream * s, size_t mdatStartOff
     return 0;
 }
 
+static avifResult avifEncoderWriteMdat(avifEncoder * encoder,
```

Comment:
Nit: Rename this `avifEncoderWriteMediaDataBox `. More consistent with the `avifParse*Box` function names in src/read.c.